pages/settings.vue

```diff
@@ -47,21 +47,21 @@
               >
               <TPopup
                 v-if="deleteAccountPopupVisible"
-                title="Confirm account deletion"
+                :title="$t('settings.account.popup.title')"
```

Comment:
There might be more popups on that page. I would be more specific, i.e. 
```suggestion
                :title="$t('settings.account.popup.delete.title')"
```